src/backend/users/models.py

```diff
@@ -2,4 +2,45 @@
 from django.contrib.auth.models import User
 
 class UserProfile(models.Model):
+    class TravelMode(models.TextChoices):
+        CAR = "CAR", "car"
+        BUS_TRAIN = "BT", "bus/train"
+        BUS_TRAIN_EXPRESS = "BTE", "bus/train/express"
+
+    class Priorities(models.TextChoices):
+        NOT_IMPORTANT = "NI", "Not important"
+        SOMEWHAT_IMPORTANT = "SI", "Somewhat important"
+        IMPORTANT = "I", "Important"
+        VERY_IMPORTANT = "VI", "Very important"
+
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    full_name = models.CharField(blank=True, max_length=200) # This is used in place of Django's auto first name and last name fields; they're still present in User but unused
+    has_voucher = models.BooleanField(default=False)
+    voucher_number = models.CharField(blank=True, max_length=10)
+    voucher_bedrooms = models.IntegerField(blank=True, null=True)
+    rent_budget = models.IntegerField(blank=True, null=True)
+    desired_bedrooms = models.IntegerField(blank=True, null=True)
+    travel_mode = models.CharField(choices=TravelMode.choices, max_length=3, default=TravelMode.BUS_TRAIN)
+    commute_priority = models.CharField(choices=Priorities.choices, max_length=2, default=Priorities.NOT_IMPORTANT)
+    school_quality_priority = models.CharField(choices=Priorities.choices, max_length=2, default=Priorities.NOT_IMPORTANT)
+    public_safety_priority = models.CharField(choices=Priorities.choices, max_length=2, default=Priorities.SOMEWHAT_IMPORTANT)
+
+    def __str__(self):
+        return self.full_name
```

Comment:
I just noticed one minor thing: because `full_name` isn't required, this can result in the string representation being an empty string if that field's not available, e.g.:
<img width="486" alt="Screen Shot 2022-04-27 at 08 37 21" src="https://user-images.githubusercontent.com/447977/165519555-523054d5-e178-42de-8d89-13448465a1b6.png">

I'd suggest using `self.user.email` instead since that's a required field.